jujugui/static/gui/src/app/components/user-profile/model-list/model-list.js

```diff
@@ -177,6 +177,7 @@ YUI.add('user-profile-model-list', function() {
           return null;
         }
       }
+      const lastConnection = model.lastConnection || '--';
```

Comment:
Is this `||` still necessary with `DateDisplay` now being able to handle invalid dates?

I'd imagine that the date would now display as "undefined" if it was, so maybe the `DateDisplay` component should handle this case internally as well.